errors.go

```diff
@@ -0,0 +1,110 @@
+package fileserver
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/avto-dev/go-simple-fileserver/cache"
+)
+
+// ErrorPageTemplate  is error page template in string representation. Is allowed to use basic "replacing patterns"
+// like `{{ code }}` or `{{ message }}`
+type ErrorPageTemplate string
+
+// String converts template into string representation.
+func (t ErrorPageTemplate) String() string { return string(t) }
+
+// Build makes registered patterns replacing.
+func (t ErrorPageTemplate) Build(errorCode int) string {
+	var out string = t.String()
```

Comment:
Why not `out := t.String()`

Reply:
agree, but in the future - use "suggestions" for this